### DISCUSSION TITLE
AppShell and Navbar - is it possible to hide the Navbar based on the route that is active?

### DISCUSSION THREAD

**Discussion by pamgaiguer** [mantinedev/mantine#2294] [Q&A]
Is there a way to, regret to the route of the page, the Appshell / Navbar appear or not? 

Under the 'location' route, the navbar **should appear**
![image](https://user-images.githubusercontent.com/1967311/187573788-1bad1146-0b10-49ea-a45e-97baa0dacf9a.png)

under the location 'who-to-call' the navbar **should be hidden**

![image](https://user-images.githubusercontent.com/1967311/187574116-133f159d-db0a-4275-9fc4-0f48cfcd7129.png)

I've tried to use the idea of location from the NavLink component (**active={location.pathname === '/home'}**) but **active** is not a valid prop from NavBar :/

Any ideas guys?!
Thank you

**Comment by rtivital** [comment]
`<AppShell navbar={location.pathname === '/home' && <Navbar />} />`

**Reply by pamgaiguer** [reply]
![image](https://user-images.githubusercontent.com/1967311/188016356-264ab2e3-05c3-479b-bd59-215edb9326c8.png)

hey @rtivital thanks for you answer, but even with mantine 5.2.0 the navbar prop doesn't accept a boolean and an Element at the same time. Could you please help me ?

Thanks

**Reply by rtivital** [reply]
Try `<AppShell navbar={location.pathname === '/home' ? <Navbar /> : undefined} />`
